scripts.js

```diff
@@ -0,0 +1,19 @@
+function daysInYear(year) {
+  try {
+    if (year < 0 || !Number.isInteger(year) || typeof year !== 'number') {
+      throw new Error('Incorrect year!');
+    }
+    const days = (new Date(year,11,31) - new Date(year,0,0))/86400000
+    return days;
+  }
+  catch {
+    console.log('Incorrect year!')
```

Comment:
Ok, but the main purpose of throwing a custom error is catching it in another function